lib/screens/login.screen.dart

```diff
@@ -13,8 +14,22 @@ class LoginState extends State<LoginScreen> {
   final GoogleSignIn _googleSignIn = GoogleSignIn(scopes: ['email']);
   final FirebaseAuth _auth = FirebaseAuth.instance;
   final FirebaseService firebaseService = FirebaseService();
+  bool showIndicator = false; 
 
+  @override
+  build(BuildContext context) => Scaffold(
+        body: showIndicator
+            ? Center(child: CupertinoActivityIndicator())
```

Comment:
Preferably this should be a`CircularProgressIndicator` or similar instead of the Cupertino one.